daemon/daemon.go

```diff
@@ -1056,7 +1058,10 @@ func writeDistributionProgress(cancelFunc func(), outStream io.Writer, progressC
 
 	for prog := range progressChan {
 		if err := progressOutput.WriteProgress(prog); err != nil && !operationCancelled {
-			logrus.Errorf("error writing progress to client: %v", err)
+			// don't log broken pipe errors as this is the normal case when a client aborts
+			if !isBrokenPipe(err) {
+				logrus.Errorf("error writing progress to client: %v", err)
```

Comment:
But what I see when 2 clients are connected and then 1 disconnects is the epipe error, but the other client continues on and the pull is not cancelled.